packages/engine/src/actions/getGlobal.js

```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import getFromObject from './getFromObject.js';
+
+function createGetGlobal({ arrayIndices, blockId, context }) {
+  return function getGlobal(params) {
+    return getFromObject({
+      arrayIndices,
+      location: blockId,
+      object: context._internal.lowdefy.lowdefyGlobal || {},
```

Comment:
lowdefyGlobal should always be defined, so the `|| {}` is not necessary. This function should error if lowdefyGlobal is undefined